Public/Src/Tools/Execution.Analyzer/Analyzers.Core/XLGPlusPlus/NewDumpPipAnalyzer.cs

```diff
@@ -0,0 +1,214 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using BuildXL.Analyzers.Core.XLGPlusPlus;
+using BuildXL.Execution.Analyzer.Xldb;
+using BuildXL.ToolSupport;
+
+namespace BuildXL.Execution.Analyzer
+{
+    internal partial class Args
+    {
+        public Analyzer InitializeNewDumpPipAnalyzer()

```

Comment:
>InitializeNewDumpPipAnalyzer [](start = 24, length = 28)

Can we refer to the "new" analyzers as {Name}XldbAnalyzer instead of New{Name}Analyzer.